### PR TITLE
PR improvements

### PR DESCRIPTION
**Pull request by Kristian-ZH**
PR improvements

### CODE REVIEW

File: fluent-bit-to-loki/README.md [hunk 1]
```diff
@@ -33,14 +31,14 @@ This plugin is implemented with [Fluent Bit's Go plugin](https://github.com/flue
...
-| ExtractKubernetesMetadataFromTag | If set the plugin will try to extract the namespace, pod_name and container_id from a tag in the log entry | `false`
-| TagKey | This is the key in the log entry which holds the tag.The tag must not be nested | "tag"
-| TagPrefix | This is the prefix of the tag. In the prefix no metadata will be searched. The prefix must not contain group expression(`()`). | none
-| TagExpression | This is the regex expression which will be used for matching the metadata. It must contains only 3 group expressions (`()`) which will be `pod name`, `namespace` and the `container id` | "\\.(.*)_(.*)_(.*)-.*\\.log"
-| DropLogEntryWithoutK8sMetadata | If set the log entries without kubernetes metadata will be dropped | `false` 
+| ReplaceOutOfOrderTS | Overwrites the timestamp of out of order records. Their timestamp will replaced with the timestamp of the last entry. | `false`
+| FallbackToTagWhenMetadataIsMissing | If set the plugin will try to extract the `namespace`, `pod_name` and `container_name` the tag when the metadata is missing | `false`
```
Comment (vpnachev) [inline fluent-bit-to-loki/README.md]:
```suggestion
| FallbackToTagWhenMetadataIsMissing | If set the plugin will try to extract the `namespace`, `pod_name` and `container_name` from the tag when the metadata is missing | `false`
```

File: fluent-bit-to-loki/README.md [hunk 2]
```diff
@@ -138,7 +177,7 @@ You can run multiple plugin instances in the same fluent-bit process, for exampl
 * gcc (for cgo)
 
 ```bash
-make fluent-bit-plugin
+make plugin
```
Comment (vpnachev) [inline fluent-bit-to-loki/README.md]: This command is not available in the root of the repo. You need one of the following

```suggestion
make -C fluent-bit-to-loki plugin
```
or
```suggestion
cd fluent-bit-to-loki
make plugin
```

**Review by vpnachev — COMMENTED** [review]
Some minor suggestions

**Review by vlvasilev — APPROVED** [review]
/lgtm

**Pull request merged**